README.md

```diff
@@ -18,18 +18,41 @@ For running the services that are event sourced, it needs a database for persist
 
 ## Locally running on microk8s:
 
-1. Install microk8s if not already installed: see https://microk8s.io/docs/install-alternatives for instructions
-2. Install scylla-operator on microk8s: see https://operator.docs.scylladb.com/stable/generic.html for instructions
-3. Run command `microk8s kubectl apply -f microApply.yaml`
-4. Check status with `microk8s kubectl get pods -o wide`
-5. Inspect pod using `microk8s kubectl describe pod [pod-name]`
-6. Inspect services using `microk8s kubectl logs [pod-name] -c [service-name]`
+### IMPORTANT NOTE
+For now, local scylla-db services can only be connected to in microk8s by changing a service's `application.conf` to use the *internal* `ClusterIP` of the `scylla-db` service
+For this, `scylla applyForInternalIP.yaml`, must be used in placy of `scyllaApply.yaml` in above steps.
+
+## Installation Instructions
+1. Install microk8s if not already installed (: see https://microk8s.io/docs/install-alternatives for instructions)
+   - If on mac, just run `brew install microk8s`
+2. Enable dns with `microk8s enable dns`
+3. Install scylla
+   1. Run command to download docker image (from https://opensource.docs.scylladb.com/stable/operating-scylla/procedures/tips/best-practices-scylla-on-docker.html)
+      - `docker run --name some-scylla -d scylladb/scylla`
+   2. Follow instructions for "Working with locally built images without a registry" using previously downloaded scylla-db docker image (from https://microk8s.io/docs/registry-images)
+      1. `docker save scylladb/scylla > scylla.tar`
+      2. `microk8s ctr image import scylladb/scylla.tar`
```

Comment:
This line is failing for me. I'm not sure where it's looking for the file, but it does not appear to be the current directory.

```dag@Dags-MacBook-Pro back-end % microk8s ctr image import scylladb/scylla.tar
ctr: open scylladb/scylla.tar: no such file or directory
An error occurred when trying to execute 'sudo microk8s.ctr image import scylladb/scylla.tar' with 'multipass': returned exit code 1.
dag@Dags-MacBook-Pro back-end % ls
DCO.txt				improving-app-gateway.tar	organization
LICENSE				improving-app-gatling.tar	project
README.md			improving-app-member.tar	scylla.tar
build.sbt			improving-app-organization.tar	scyllaApply.yaml
busyApply.yaml			improving-app-store.tar		scyllaApplyForInternalIP.yaml
common-types			improving-app-tenant.tar	store
customer			inventory			target
docker-compose.yml		it				tenant
gateway				member
gatling				microApply.yaml
dag@Dags-MacBook-Pro back-end % microk8s ctr image import scylla.tar
ctr: open scylla.tar: no such file or directory
An error occurred when trying to execute 'sudo microk8s.ctr image import scylla.tar' with 'multipass': returned exit code 1.
dag@Dags-MacBook-Pro back-end % microk8s ctr image import ./scylla.tar
ctr: open ./scylla.tar: no such file or directory
An error occurred when trying to execute 'sudo microk8s.ctr image import ./scylla.tar' with 'multipass': returned exit code 1.
```

Reply:
I made a mistake - it should have read `microk8s ctr image import scylla.tar`. Updating

Reply:
that makes more sense, however: 
```
dag@Dags-MacBook-Pro back-end % microk8s ctr image import scylla.tar
ctr: open scylla.tar: no such file or directory
An error occurred when trying to execute 'sudo microk8s.ctr image import scylla.tar' with 'multipass': returned exit code 1.
dag@Dags-MacBook-Pro back-end % ls scylla.tar
scylla.tar
```

Reply:
something peculiar is going on:
```
dag@Dags-MacBook-Pro back-end % pwd
/Users/dag/work/improving-app/back-end
dag@Dags-MacBook-Pro back-end % microk8s ctr image import /Users/dag/work/improving-app/back-end/scylla.tar
ctr: open /Users/dag/work/improving-app/back-end/scylla.tar: no such file or directory
An error occurred when trying to execute 'sudo microk8s.ctr image import /Users/dag/work/improving-app/back-end/scylla.tar' with 'multipass': returned exit code 1.
dag@Dags-MacBook-Pro back-end %
```